server/miljø.ts

```diff
@@ -51,19 +48,30 @@ const lokaltMiljø: Environment = lagMiljø({
     ? 'https://familie-ef-soknad-api.intern.dev.nav.no/familie/alene-med-barn/soknad-api/api'
     : 'http://localhost:8091/api',
   oAuthCallbackUri: 'https://localhost:8080/familie/alene-med-barn/minside/oauth2/callback',
+  ettersendingUrl: 'https://familie.ekstern.dev.nav.no/familie/alene-med-barn/ettersending',
+  søknadOvergangsstønadUrl: 'https://familie.ekstern.dev.nav.no/familie/alene-med-barn/soknad/',
+  søknadBarnetilsynUrl:
+    'https://familie.ekstern.dev.nav.no/familie/alene-med-barn/soknad/barnetilsyn',
+  søknadSkolepengerUrl:
+    'https://familie.ekstern.dev.nav.no/familie/alene-med-barn/soknad/skolepenger',
+  klageUrl: 'https://klage.intern.dev.nav.no/',
 });
 
-const devMiljø: Environment = lagMiljø({});
-
-const prodMiljø: Environment = lagMiljø({
-  minSideUrl: 'https://www.nav.no/minside/',
-  oAuthCallbackUri: 'https://www.nav.no/familie/alene-med-barn/minside/oauth2/callback',
-  ettersendingUrl: 'https://www.nav.no/familie/alene-med-barn/ettersending',
-  søknadOvergangsstønadUrl: 'https://www.nav.no/familie/alene-med-barn/soknad',
-  søknadBarnetilsynUrl: 'https://www.nav.no/familie/alene-med-barn/soknad/barnetilsyn',
-  søknadSkolepengerUrl: 'https://www.nav.no/familie/alene-med-barn/soknad/skolepenger',
+const devMiljø: Environment = lagMiljø({
+  minSideUrl: 'https://www.intern.dev.nav.no/minside/',
```

Comment:
Hvis vi bytter plass på devMiljø og lokaltMiljø så kan du si noe som dette:
```
const lokaltMiljø = lagMiljø({
...devMiljø, 
  port: 3000,
  søknadApiProxyUrl: brukDevApi()
    ? 'https://familie-ef-soknad-api.intern.dev.nav.no/familie/alene-med-barn/soknad-api/api'
    : 'http://localhost:8091/api',
  oAuthCallbackUri: 'https://localhost:8080/familie/alene-med-barn/minside/oauth2/callback', 
})
```

Vet ikke om det er bedre, men slipper isåfall å gjenta for mye

Reply:
Fint med mindre duplikat kode. Tenker at koden fortsatt er lesbar